modules/hcp-topology-spread-constraint.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-topology-spread-constraint_{context}"]
+= Spreading node pool VMs by using topologySpreadConstraint
+
+By default, KubeVirt virtual machines (VMs) created by a node pool are scheduled to any available nodes with the capacity to run the VM. By default, the `topologySpreadConstraint` constraint is set to schedule VMs on multiple nodes.
+
+In some scenarios, VMs of a node pool might run on the same node which can cause an availability issue. To avoid distribution of VMs on a single node, use the descheduler to continuously honour the `topologySpreadConstraint` constraint to spread VMs on multiple nodes.
+
+.Prerequisites
+
+* You installed the {descheduler-operator}. For more information, see "Installing the descheduler".
+
+.Procedure
+
+* Modify the `KubeDescheduler` custom resource (CR) to use the `SoftTopologyAndDuplicates` profile so that you maintain the `topologySpreadConstraint` constraint settings. Run the following command:
++
+[source,terminal]
+----
+$ oc edit kubedescheduler cluster -n openshift-kube-descheduler-operator
+----
++
+The `KubeDescheduler` CR named `cluster` runs in the `openshift-kube-descheduler-operator` namespace.
```

Comment:
Consider moving "The `KubeDescheduler` CR named `cluster` runs in the `openshift-kube-descheduler-operator` namespace." above the command.